src/probe/probe.c

```diff
@@ -3,10 +3,26 @@
 // Copyright(c) 2019 Intel Corporation. All rights reserved.
 //
 // Author: Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+// Author: Adrian Bonislawski <adrian.bonislawski@linux.intel.com>
 
+#include <config.h>
+
+#if CONFIG_PROBE
```

Comment:
I assume we can't trust cmake to not compile this file at all when the config is disabled?

Reply:
Well, lets trust cmake then :)